src/apps/answers/service.py

```diff
@@ -982,14 +1004,22 @@ async def get_flow_submissions(
 
         return FlowSubmissionsDetails(submissions=submissions, flows=flows), total
 
-    async def get_assessments_count(self, answer_ids: list[uuid.UUID]) -> dict[uuid.UUID, ReviewsCount]:
+    async def get_answer_assessments_count(self, answer_ids: list[uuid.UUID]) -> dict[uuid.UUID, ReviewsCount]:
         answer_reviewers_t = await AnswerItemsCRUD(self.answer_session).get_reviewers_by_answers(answer_ids)
         answer_reviewers: dict[uuid.UUID, ReviewsCount] = {}
         for answer_id, reviewers in answer_reviewers_t:
             mine = 1 if self.user_id in reviewers else 0
             answer_reviewers[answer_id] = ReviewsCount(mine=mine, other=len(reviewers) - mine)
         return answer_reviewers
 
+    async def get_submission_assessment_count(self, submission_ids: list[uuid.UUID]) -> dict[uuid.UUID, ReviewsCount]:
+        answer_reviewers_t = await AnswerItemsCRUD(self.answer_session).get_reviewers_by_submission(submission_ids)
```

Comment:
What does `_t` posix mean?

Reply:
This means that the variable contains a complex type